From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@ClarityQueen__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (v-diree). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

carolinacmoravia committed:
C52849: Broken bookmark: [Setup your C++ application]

@aasthan, 
Could you review the reported bookmark? It is not working.
Many thanks for your time in advance.

iainfoulds committed:
Doc outline

iainfoulds committed:
In-progress updates

Commit:
drafting

Commit:
safekeeping

iainfoulds committed:
Fleshing out. Splitting to include file. Adding OS versions

iainfoulds committed:
Edits and revisions

Commit:
safekeeping

iainfoulds committed:
Updates and revisions

iainfoulds committed:
Acrolinx updates

iainfoulds committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into vmmvcphase2-infrastructureautomation

Pulling in master after VM's how-to refactoring.

v-royhar committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master'

carolinacmoravia committed:
M8268: Possible incorrect string: "Web & Mobile Services"

@jeffhollan, 
It seems that it should read "Web + Mobile" instead of "Web & Mobile Services."
Thank you.

Commit:
safekeeping

dlepow committed:
Very rough content merge and cut

davidmu1 committed:
new article

davidmu1 committed:
incorp acrolinx

dlepow committed:
Work in progress

dlepow committed:
Work in progress

davidmu1 committed:
corrections

dlepow committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into hpcconcept

dlepow committed:
work in progress

dlepow committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into hpcconcept

dlepow committed:
work in progress

dlepow committed:
first draft complete

dlepow committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into hpcconcept

dlepow committed:
link fixes

dlepow committed:
link fixes

dlepow committed:
tweaks

dlepow committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into hpcconcept

dlepow committed:
Redirecting old articles

dlepow committed:
Retiring/redirecting old articles

iainfoulds committed:
Staging branch for MVC conceptual docs + TOC refactoring

iainfoulds committed:
Refactor availability concepts doc

iainfoulds committed:
Update virtual-machines-common-regions-and-availability.md

davidmu1 committed:
new article

davidmu1 committed:
incorp acrolinx

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into backup-recovery

 Please enter a commit message to explain why this merge is necessary,

Commit:
corrections

iainfoulds committed:
Updating Win ARM link

iainfoulds committed:
Merge branch 'vmmvcphase2-conceptualstaging' of https://github.com/iainfoulds/azure-docs-pr into vmmvcphase2-conceptualstaging

iainfoulds committed:
Updating Windows sizes link

Commit:
corrections

iainfoulds committed:
Moving Windows network overview to include, switching out Linux version

Commit:
drafting

dlepow committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into hpcconcept

dlepow committed:
Acrolinx tweaks

Commit:
creating include

Commit:
fixing links

Commit:
incorp feedback

iainfoulds committed:
Updating links and media files

iainfoulds committed:
Merge branch 'master' into vmmvcphase2-conceptualstaging

iainfoulds committed:
Incorporating peer feedback

iainfoulds committed:
Updating links

iainfoulds committed:
Updating links and artwork

iainfoulds committed:
Merge branch 'vmmvcphase2-conceptualstaging' of https://github.com/iainfoulds/azure-docs-pr into vmmvcphase2-conceptualstaging

dlepow committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into hpcconcept

dlepow committed:
review comments

dlepow committed:
fixed link

iainfoulds committed:
Working draft of new conceptual doc for containers in Azure

iainfoulds committed:
Incorporating peer feedback

iainfoulds committed:
OS metadata update

iainfoulds committed:
Acrolinx updates

iainfoulds committed:
Incorporating feedback. Moving sections around and expanding wording.

iainfoulds committed:
Adding intro text and next steps

iainfoulds committed:
Updating H1s

iainfoulds committed:
Incorporating Ralph's feedback on opening section

iainfoulds committed:
Merge pull request #7 from cynthn/backup-recovery

Backup\Recovery Overview

iainfoulds committed:
Merge pull request #8 from davidmu1/autoscale

Autoscale

iainfoulds committed:
Merge pull request #9 from davidmu1/securemonitor

Securemonitor

iainfoulds committed:
Merge pull request #10 from dlepow/hpcconcept

HPC Conceptual Doc

iainfoulds committed:
Merge pull request #12 from iainfoulds/vmmvcphase2-infrastructureautomation

Vmmvcphase2 infrastructureautomation

iainfoulds committed:
Merge pull request #13 from iainfoulds/vmmvcphase2-containers

Vmmvcphase2 containers

Lisaco88 committed:
edit pass: application-gateway-create-url-route-arm-ps

iainfoulds committed:
Updating TOC links

iainfoulds committed:
Merge branch 'vmmvcphase2-conceptualstaging' of https://github.com/iainfoulds/azure-docs-pr into vmmvcphase2-conceptualstaging

Lisaco88 committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into application-gateway-create-url-route-arm-ps

Lisaco88 committed:
edit pass: application-gateway-create-url-route-arm-ps

Lisaco88 committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into application-gateway-create-url-route-arm-ps

Lisaco88 committed:
edit pass: application-gateway-create-url-route-arm-ps

Lisaco88 committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into application-gateway-create-url-route-arm-ps

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-master

Lisaco88 committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into application-gateway-create-url-route-arm-ps

icommit.

Lisaco88 committed:
edit pass: application-gateway-create-url-route-arm-ps

Lisaco88 committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into application-gateway-create-url-route-arm-ps

iainfoulds committed:
Incorporating CDA feedback

iainfoulds committed:
Fixing merge conflict

Commit:
update Service Fabric landing page

Commit:
moved paragraph type

Commit:
update QS section

Commit:
removed en-us

neilpeterson committed:
tutorial updates

neilpeterson committed:
updated doc

neilpeterson committed:
doc updates

neilpeterson committed:
updates

Commit:
Update with latest version, important info.

neilpeterson committed:
Merge remote-tracking branch 'upstream/master' into acs-tutorial-updates

Commit:
Acrolinx suggestions.

Commit:
More Acrolinx

neilpeterson committed:
updates

Commit:
More Acrolinx

neilpeterson committed:
updates

neilpeterson committed:
updates

neilpeterson committed:
Merge branch 'master' into acs-tutorial-updates

Commit:
Acrolinx grammar feedback

neilpeterson committed:
updates

neilpeterson committed:
Merge branch 'acs-tutorial-updates' of https://github.com/neilpeterson/azure-docs-pr into acs-tutorial-updates

tdykstra committed:
minor updates

tdykstra committed:
minor updates

neilpeterson committed:
removed note

Commit:
Minor text fix

tdykstra committed:
change prereqs

tdykstra committed:
typo

neilpeterson committed:
updates

neilpeterson committed:
updates

v-viinde committed:
USER STORY 1078094 SaaS App Tutorial: QPrism

v-viinde committed:
Updated

v-viinde committed:
Updated

Ckarst committed:
Update sql-data-warehouse-load-polybase-guide.md

Added performance optimizations and limitations.

Ckarst committed:
Update sql-data-warehouse-load-polybase-guide.md

Fixed bad bullets

ShawnJackson committed:
Update application-gateway-create-url-route-arm-ps.md

ShawnJackson committed:
Update application-gateway-create-url-route-arm-ps.md

Commit:
update lynda logo

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-master

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-master

HeidiSteen committed:
blob data source list in an include

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-master

HeidiSteen committed:
edited to raise acro score above 80, fixed broken links

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-master

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-master

neilpeterson committed:
updates

iainfoulds committed:
Pulling master and fixing merge conflict

HeidiSteen committed:
edits

HeidiSteen committed:
Janusz edit for a non-blob-related article

LizCasey committed:
Updates that align with Azure VS 2017 project requirements.

iainfoulds committed:
Incorporating PM feedback

xujing-ms committed:
Update hybrid-use-benefit-licensing.md

neilpeterson committed:
new command to return login server

iainfoulds committed:
Splitting security and monitoring in to two separate docs, per PM feedback

Commit:
Fixing language

Commit:
unbolding file name

cynthn committed:
Merge branch 'vmmvcphase2-conceptualstaging' into backup-recovery

cynthn committed:
Merge branch 'vmmvcphase2-conceptualstaging' into backup-recovery

mimig1 committed:
Fixed bookmark

iainfoulds committed:
Final upstream pull and merge conflict fixes

iainfoulds committed:
Merge pull request #14 from cynthn/backup-recovery

Backup recovery updates

BrucePerlerMS committed:
update example formatting

BrucePerlerMS committed:
update for Frank's review

neilpeterson committed:
updates

neilpeterson committed:
acrolinx

tfitzmac committed:
revising manage with cli

tfitzmac committed:
added overview diagram

Commit:
imagefix01

mimig1 committed:
Cosmos DB: Removed quotes from meta description.

tdykstra committed:
Azure in title

tdykstra committed:
Merge branch 'wjvs' of https://github.com/tdykstra/azure-docs-pr into wjvs

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-master

HeidiSteen committed:
Lucene query JSFiddle example

BrucePerlerMS committed:
formatting and explanation updates

v-royhar committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master'

neilpeterson committed:
updates

v-royhar committed:
Add export-your-model images and folder

BrucePerlerMS committed:
acrolinx feedback

BrucePerlerMS committed:
more acrolinx feedback

tfitzmac committed:
revised manage cli

BrucePerlerMS committed:
acronym expansion

BrucePerlerMS committed:
tweak

BrucePerlerMS committed:
formatting

BrucePerlerMS committed:
more formatting

BrucePerlerMS committed:
formatting

raiye committed:
Update cloud-services-guestos-update-matrix.md

Commit:
tocfix

Commit:
update for Microsoft account support

HeidiSteen committed:
10k is now 5 k char as measured by string length, keyword is now key phrase

PRMerger10 committed:
Merge pull request #22696 from raiye/patch-39

Update cloud-services-guestos-update-matrix.md

BrucePerlerMS committed:
resolve merge conflicts

tfitzmac committed:
fix code

PRMerger10 committed:
Merge pull request #22698 from stevestein/SqlDbAadEdit

update for Microsoft account support

PRMerger19 committed:
Merge pull request #22700 from xujing-ms/master

Update hybrid-use-benefit-licensing.md

BrucePerlerMS committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into brp-work

PRMerger13 committed:
Merge pull request #19369 from carolinacmoravia/patch-334

M8268: Possible incorrect string: "Web & Mobile Services"

tonguyen10 committed:
Update billing-getting-started.md

tonguyen10 committed:
Delete billing-getting-started.experimental.md

ShawnJackson committed:
Merge pull request #20616 from Lisaco88/application-gateway-create-url-route-arm-ps

edit pass: application-gateway-create-url-route-arm-ps

ktoliver committed:
Merge pull request #22701 from MarkusVi/tocfix

tocfix

v-thepet committed:
links

ktoliver committed:
Merge pull request #22699 from HeidiSteen/heidist-ta-api

10k is now 5 k char as measured by string length, keyword is now key …

ktoliver committed:
Merge pull request #22681 from MarkusVi/imagefix01

imagefix01

itechedit committed:
Merge pull request #22005 from neilpeterson/acs-tutorial-updates

tutorial updates

ktoliver committed:
Merge pull request #22676 from tfitzmac/0914egimage

added overview diagram

Commit:
Merge pull request #22706 from v-thepet/newlink

production bug: links

TerryLanfear committed:
updates

TerryLanfear committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into sept-onboard

PRMerger18 committed:
Merge pull request #16501 from carolinacmoravia/patch-257

C52849: Broken bookmark: [Setup your C++ application]

TerryLanfear committed:
updates

TerryLanfear committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into sept-mig

itechedit committed:
Merge pull request #19765 from iainfoulds/vmmvcphase2-conceptualstaging

VM MVC phase 2 - new conceptual docs staging

Lisaco88 committed:
Merge pull request #22261 from BorisB2015/master

Update RHUI article.

itechedit committed:
Merge pull request #22432 from tdykstra/wjvs

minor updates to WebJobs deploy using VS doc

Lisaco88 committed:
Merge pull request #22710 from TerryLanfear/sept-onboard

Sept onboard

Lisaco88 committed:
Merge pull request #22716 from TerryLanfear/sept-mig

Sept mig

HJToland3 committed:
content tweaks

HJToland3 committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into localworking

itechedit committed:
Merge pull request #22543 from v-viinde/qprism

USER STORY 1078094 SaaS App Tutorial: QPrism

HJToland3 committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into localworking

itechedit committed:
Merge pull request #21753 from v-savila/1074718

update Service Fabric landing page

HJToland3 committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into localworking

Lisaco88 committed:
Merge pull request #22690 from v-royhar/v-royhar-export-your-model

V royhar export your model

ktoliver committed:
Merge pull request #22565 from Ckarst/patch-13

Update sql-data-warehouse-load-polybase-guide.md

Lisaco88 committed:
Merge pull request #22674 from tfitzmac/0913cli

revising manage with cli

gbowerman committed:
Adding reference to VMSS MSI support

gbowerman committed:
Explicitly mention VMSS support

Lisaco88 committed:
Merge pull request #22703 from tonguyen10/master

Update prevent unexpected charges

v-thepet committed:
Merge pull request #22633 from LizCasey/v-elicas0907

Updates that align with Azure VS 2017 project requirements.

PRMerger15 committed:
Merge pull request #22682 from MicrosoftDocs/mimig1-patch-4

Cosmos DB: Removed quotes from meta description.

PRMerger18 committed:
Merge pull request #22742 from gbowerman/patch-12

Adding reference to VMSS MSI support

rjagiewich committed:
Merge pull request #22736 from HJToland3/localworking

Localworking

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

Lisaco88 committed:
Merge pull request #22661 from BrucePerlerMS/brp-work

update example formatting

PRMerger15 committed:
Merge pull request #22744 from gbowerman/patch-13

Explicitly mention VMSS support

v-thepet committed:
Merge pull request #22613 from HeidiSteen/heidist-master

Blob data source list in an include

v-alje committed:
Merge pull request #22750 from MicrosoftDocs/FromPublicRepo

From public repo
